.github/coordinator/adapters/codex-cli.mjs

```
@@ -0,0 +1,559 @@
+    fs.closeSync(fd);
+    fd = undefined;
+    fs.linkSync(tempPath, finalPath);
+  } catch (error) {
+    if (error?.code === "EEXIST") {
+      const winner = readDurableSession({ stateDir, attempt_id, target_sha });
+      if (winner === thread_id) return finalPath;
+    }
+    throw error;
+  } finally {
+    if (fd !== undefined) fs.closeSync(fd);
+    try { fs.unlinkSync(tempPath); } catch (error) { if (error?.code !== "ENOENT") throw error; }
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)